src/python/sdk/econia_sdk/view/market.py

```diff
@@ -0,0 +1,255 @@
+    result = int(view.get_returns("market", "get_CANCEL_MAKER")[0])
+    return result
+
+def get_CANCEL_TAKER(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_CANCEL_TAKER")[0])
+    return result
+
+def get_FILL_OR_ABORT(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_FILL_OR_ABORT")[0])
+    return result
+
+def get_HI_PRICE(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_HI_PRICE")[0])
+    return result
+
+def get_IMMEDIATE_OR_CANCEL(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_IMMEDIATE_OR_CANCEL")[0])
+    return result
+
+def get_MAX_POSSIBLE(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_MAX_POSSIBLE")[0])
+    return result
+
+def get_NO_CUSTODIAN(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_NO_CUSTODIAN")[0])
+    return result
+
+def get_NO_RESTRICTION(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_NO_RESTRICTION")[0])
+    return result
+
+def get_NO_UNDERWRITER(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_NO_UNDERWRITER")[0])
+    return result
+
+def get_POST_OR_ABORT(view: EconiaViewer) -> int:
+    result = int(view.get_returns("market", "get_POST_OR_ABORT")[0])
+    return result
+
+def get_PERCENT(view: EconiaViewer) -> int:
+    result = bool(view.get_returns("market", "get_PERCENT")[0])
+    return result
+
+def get_SELL(view: EconiaViewer) -> int:
+    result = bool(view.get_returns("market", "get_SELL")[0])
+    return result
+
+def get_TICKS(view: EconiaViewer) -> int:
+    result = bool(view.get_returns("market", "get_TICKS")[0])
+    return result
+
+def get_market_order_id_counter(view: EconiaViewer, market_order_id: int) -> int:
+    returns = view.get_returns(
+        "market",
+        "get_market_order_id_counter",
+        [],
+        [str(market_order_id)]
+    )
+    return int(returns[0])
+
+def get_market_order_id_price(view: EconiaViewer, market_order_id: int) -> int:
+    returns = view.get_returns(
+        "market",
+        "get_market_order_id_price",
+        [],
+        [str(market_order_id)]
+    )
+    return int(returns[0])
+
+def get_posted_order_id_side(view: EconiaViewer, market_order_id: int) -> bool:
+    returns = view.get_returns(
+        "market",
+        "get_posted_order_id_side",
+        [],
+        [str(market_order_id)]
+    )
+    return bool(returns[0])
+
+def get_open_order(view: EconiaViewer, market_id: int, market_order_id: int) -> Optional[dict]:
+    returns = view.get_returns(
+        "market",
+        "get_open_order",
+        [],
+        [
+          str(market_id),
+          str(market_order_id)
+        ]
+    )
+    opt_val = returns[0]['vec']
+    if len(opt_val) == 0:
+        return None
+    else:
+        return _convert_open_order_value(opt_val[0])
+
+HI_64 = 18446744073709551615
```

Comment:
This would be more appropriate to have at the top of a file in a constants section or in a separate util file

Reply:
Renamed to `_HI_64` and moved to top of file